### ISSUE TITLE
Accuracy curve?

### ISSUE THREAD

**Issue by ogencoglu** [isaacgerg/keras_mnist_cae#1]
What does the accuracy curve represent here in your images? Isn't this just representation learning?

**Comment by isaacgerg** [comment]
Yes, so I would think its (actual - desired) / desired.

**Comment by ogencoglu** [comment]
But there is no classification here. Am I missing something?

**Comment by isaacgerg** [comment]
correct, no classification, but reconstruction.

**Comment by ogencoglu** [comment]
OK so what exactly does it plot in the "Acc vs. Epoch" curve? I understand the loss curve (left one). That represents accuracy of reconstruction. What about the right one?

**Comment by isaacgerg** [comment]
Its the (actual value - target value) / target_value

**Comment by ogencoglu** [comment]
Actual value is an image (a matrix), target value is an image (another matrix). The difference is yet another matrix. What does it mean to divide two matrices? Element by element? Then what? I'm not sure if "accuracy" field makes sense in this application.

**Comment by isaacgerg** [comment]
It may not.  I am just using keras.  I would post over there.  I am
assuming its the average error.

**Comment by ogencoglu** [comment]
OK, cheers.
